crontab_sample

```diff
@@ -1 +1 @@
-*/30 * * * *  flock -n /tmp/szuru-toolkit.lock -c "cd /szurubooru-toolkit && /usr/local/bin/auto-tagger \"-source:*donmai* -source:*gelbooru* -source:*yande* sort:date,asc\" >/proc/1/fd/1 2>&1"
+*/30 * * * *  flock -n /tmp/szuru-toolkit.lock -c "cd /szurubooru-toolkit && /usr/local/bin/szuru-toolkit auto-tagger \" -source:*donmai* -source:*gelbooru* -source:*yande* sort:date,asc\" >/proc/1/fd/1 2>&1"
```

Comment:
I believe that starting the query with a `-` doesn't work anymore. Could you check? Otherwise the query has to be separated by a `--` from the options. I added a note for that in the lastest Readme.